### DISCUSSION TITLE
not able to build

### DISCUSSION THREAD

**Discussion by JacksReacher** [Megabit/Blazorise#4822] [General]
Get a fresh clone and load up the solution in VS 2022 Enterprise and get a bunch of errors on build like: 
![image](https://github.com/Megabit/Blazorise/assets/8247176/35690a13-b3da-4503-9459-997f0da8cacf)

**Comment by David-Moreira** [comment]
Well the error is self explanatory... Unable to restore packages. Apparently only dev express source is configured at the moment. Please configure it so you have a source to get regular packages, I.e Microsofts nuget.

**Comment by JacksReacher** [comment]
Doh! Thanks man... forest for the trees
________________________________
